packages/portal-proto/src/features/cohortBuilder/Modals/ImportCohortModal.tsx

```diff
@@ -52,7 +52,9 @@ const SubmitButton: React.FC<SubmitButtonProps> = ({
 
       <DarkFunctionButton
         disabled={disabled}
-        onClick={() => createSet({ values: ids })}
+        onClick={() =>
+          createSet({ values: ids, intent: "user", set_type: "frozen" })
+        }
```

Comment:
<img width="1038" alt="Screenshot 2024-04-05 at 2 45 20 PM" src="https://github.com/NCI-GDC/gdc-frontend-framework/assets/4624053/06e0e2db-79c3-40ac-a82c-c3088fb0f2c3">

Reply:
"portal" on grounds that user just wants a cohort of stuff. it's us who made the stuff into a set.